doc/developer/bgp-typecodes.rst

```diff
@@ -17,9 +17,6 @@ various BGP RFCs. In the code these are defined as BGP_ATTR_<ATTR>.
 | 8     | COMMUNITIES      | [RFC 1997]                               |
 | 9     | ORIGINATOR_ID    | [RFC 4456]                               |
 | 10    | CLUSTER_LIST     | [RFC 4456]                               |
-| 11    | DPA              | [draft-ietf-idr-bgp-dpa-05.txt(expired)] |
```

Comment:
Any value in commenting these out, so it's clear that they _were_ once present? Or is it well-known that there are gaps in the assigned code values?